### PR TITLE
🐛 handle multiple-wallet issue

### PR DESCRIPTION
**Pull request by hlminh2000**
requires this change from polkadot-js: https://github.com/polkadot-js/extension/pull/685

### CODE REVIEW

**Comment by remon-nashid** [comment]
@hlminh2000 could you please add a scenario that reproduces the issue, so we could verify the fix? Thanks

File: packages/core/src/utils.ts [hunk 1]
```diff
@@ -21,15 +22,16 @@ export function isPolyMessage (message: string): boolean {
   return isPolyMessage;
 }
 
-export function observeAccounts (cb: (accounts: KeyringAccountData[]) => void) {
+export function observeAccounts (cb: (accounts: KeyringAccountData[]) => void): Subscription {
```
Comment (hlminh2000) [inline packages/core/src/utils.ts:25]: this and the first `import` statement are the only real change to this file, the rest being linter stuff

**Comment by hlminh2000** [comment]
@remon-nashid, so this happens quite consistently on the issuer, dashboard, sto-investor and governance for me... Basically just need to have a dot wallet installed together with the polymesh wallet, authorizing both wallets as they request when connecting. Observing the background logs of both wallets would show the DOT wallet responding to our messages with errors

**Review by remon-nashid — APPROVED** [review]
Looks good!

**Pull request merged**